packages/sanity/src/structure/useStructureToolSetting.ts

```diff
@@ -9,36 +10,29 @@ export function useStructureToolSetting<ValueType>(
   key: string,
   defaultValue?: ValueType,
 ): [ValueType | undefined, (_value: ValueType) => void] {
-  const settingsStore = useSettingsStore()
+  const KVStore = useKVStore()
   const [value, setValue] = useState<ValueType | undefined>(defaultValue)
 
-  const structureToolSettings = useMemo(
-    () => settingsStore.forNamespace('structure-tool'),
-    [settingsStore],
-  )
+  const KVStoreKey = namespace ? `structure-tool::${namespace}::${key}` : `structure-tool::${key}`
```

Comment:
We're having internal comments about namespacing, so I don't expect this string interpolation to be permanent, hopefully